pyproject.toml

```diff
@@ -11,11 +11,11 @@ packages = [
 ]
 
```

Comment:
Hmm. There are no classifiers here. I guess we don't have to fix that now. It's of questionable value anyway. But I noticed because I was checking to see if they were updated.